frontend/styles/global.css

```diff
@@ -51,3 +52,40 @@ p {
   background: #66f28d !important;
   height: 4px !important;
 }
+
+/* noto-sans-hk-regular - latin */
+@font-face {
+  font-family: 'Noto Sans HK';
+  font-style: normal;
+  font-weight: 400;
+  src: local('Noto Sans HK Regular'), local('NotoSansHK-Regular'),
+       url('../fonts/noto-sans-hk-v5-latin-regular.woff2') format('woff2'), /* Chrome 26+, Opera 23+, Firefox 39+ */
```

Comment:
we can move the `fonts` folder inside the `public/static` path where we server all of our static files.
and change the path
Ex : ` url('../public/static/fonts/noto-sans-hk-v5-latin-regular.woff2') format('woff2'),`